src/app/components/shared/header-illustrasjon/HeaderIllustrasjon.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+import { Sidetittel } from 'nav-frontend-typografi';
+
+import CustomSVG from '../custom-svg/CustomSVG';
+import SpeechBubble from '../speech-bubble/SpeechBubble';
+
+import './headerIllustration.less';
+
+const HeaderIllustration = ({ title, dialog, svg }) => (
+	<div className="headerIllustration">
+		<div className="headerIllustration__title">
+			<Sidetittel>{title}</Sidetittel>
+		</div>
+		<div className="headerIllustration__speechBubble">
+			<SpeechBubble title={dialog.title} text={dialog.text} />
```

Comment:
Her vil vi vel potensielt få feil i konsollet (... text/title of null) hvis `dialog` ikke er angitt og settes til null by default. Muligens vil det samme problemet oppstå internt i CustomSVG for `svg`. Bedre å ha `.isRequired` på de to proptypene her?